source/renderer/app/components/status/NetworkStatus.js

```diff
@@ -248,202 +486,292 @@ export default class NetworkStatus extends Component<Props, State> {
+                  }
+                >
+                  {remainingUnsyncedBlocks > 0 ? remainingUnsyncedBlocks : '-'}
                 </td>
               </tr>
               <tr>
-                <td>latestLocalBlockAge:</td>
-                <td className={latestLocalBlockAgeClasses}>
+                <td>{intl.formatMessage(messages.latestLocalBlockAge)}:</td>
+                <td
+                  className={latestLocalBlockAgeClasses}
+                  title={
+                    latestLocalBlockTimestamp > 0
+                      ? `${latestLocalBlockAge} ms`
+                      : '-'
+                  }
+                >
                   {latestLocalBlockTimestamp > 0
                     ? `${latestLocalBlockAge} ms`
                     : '-'}
                 </td>
               </tr>
               <tr>
-                <td>latestNetworkBlockAge:</td>
-                <td className={latestNetworkBlockAgeClasses}>
+                <td>{intl.formatMessage(messages.latestNetworkBlockAge)}:</td>
+                <td
+                  className={latestNetworkBlockAgeClasses}
+                  title={
+                    latestNetworkBlockTimestamp > 0
+                      ? `${latestNetworkBlockAge} ms`
+                      : '-'
+                  }
+                >
                   {latestNetworkBlockTimestamp > 0
                     ? `${latestNetworkBlockAge} ms`
                     : '-'}
                 </td>
               </tr>
               <tr>
-                <td>localTimeDifference:</td>
+                <td>{intl.formatMessage(messages.localTimeDifference)}:</td>
                 <td>
-                  <span className={localTimeDifferenceClasses}>
-                    {isNTPServiceReachable
-                      ? `${localTimeDifference || 0} μs`
-                      : 'NTP service unreachable'}
-                  </span>{' '}
-                  |&nbsp;
                   <button
                     onClick={() => onForceCheckLocalTimeDifference()}
                     disabled={isForceCheckingNodeTime || !isConnected}
                   >
-                    {isForceCheckingNodeTime ? 'Checking...' : 'Check time'}
+                    {isForceCheckingNodeTime
+                      ? intl.formatMessage(messages.localTimeDifferenceChecking)
+                      : intl.formatMessage(
+                          messages.localTimeDifferenceCheckTime
+                        )}
                   </button>
+                  <span
+                    className={localTimeDifferenceClasses}
+                    title={
+                      isNTPServiceReachable
+                        ? `${localTimeDifference || 0} μs`
+                        : 'NTP service unreachable'
+                    }
+                  >
+                    {isNTPServiceReachable
+                      ? `${localTimeDifference || 0} μs`
+                      : 'NTP service unreachable'}
```

Comment:
@DeeJayElly please add a translation for this 🙏

Reply:
@nikolaglumac done